pkgs/development/python-modules/pslab/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, buildPythonPackage, fetchPypi, numpy, pyserial, scipy }:
+
+buildPythonPackage rec {
+  pname = "pslab";
+  version = "2.4.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0iifjz7j6g982sc1id91d2krrv6zr42v2fzln9kw7nwjzg1j2j3a";
+  };
+
+  # Tests not included in the pypi archive
```

Comment:
Please fetch from github.

Reply:
They have broken tests upstream afaiu. It's a bit annoying. I'll have to patch to make (some) tests go.

Reply:
I don't know how to blacklists the python files that fail to load on "setup.py test". I asked upstream how they run the tests. They have in some files even old names for their own libs (PSL instead of pslab...).

I can download from github but I'll have to write "Upstream seems not to maintain the tests" and set doCheck = false. I don't want to fix their tests to get this merged, if possible.

Reply:
I learnt about tox with help from the package developers. Finally the software tests run.